README.md

```diff
@@ -10,8 +11,15 @@
     - [State and storage](#state-and-storage)
     - [Routing](#routing)
     - [Testing](#testing)
+    - [Localization](#localization)
     - [Collection of scripts](#collection-of-scripts)
 
+### Setup
+
+1. `npm run create-env` to create your own development and production .env files
+2. `yarn install`
+3. `npm start`
```

Comment:
use yarn start instead. we should use yarn tool everywhere

Reply:
Resolved.